.github/workflows/ci.yml

```diff
@@ -75,7 +75,7 @@ jobs:
 
       - name: Release
         if: ${{ matrix.binary == 'release' }}
-        run: cargo build --release
+        run: cargo build --locked --release
 
```

Comment:
`--locked` triggers a compiler error if the build would update the lock file.

Reply:
good one